src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    protected static Random random = new Random();
+
+    public Colors getRandomColor() {
```

Comment:
There is a redundant empty line before this method. According to the checklist, please avoid adding redundant empty lines after method signatures.